publication-commons/src/main/java/no/unit/nva/publication/model/storage/GeneralSupportRequestDao.java

```diff
@@ -0,0 +1,74 @@
+package no.unit.nva.publication.model.storage;
+
+import static no.unit.nva.publication.storage.model.DatabaseConstants.KEY_FIELDS_DELIMITER;
+import com.amazonaws.services.dynamodbv2.model.TransactWriteItemsRequest;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import java.net.URI;
+import no.unit.nva.commons.json.JsonSerializable;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.model.business.Entity;
+import no.unit.nva.publication.model.business.GeneralSupportRequest;
+
+@JsonTypeName(GeneralSupportRequestDao.TYPE)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class GeneralSupportRequestDao extends TicketDao implements JsonSerializable {
+    
+    public static final String JOIN_BY_RESOURCE_INDEX_ORDER_PREFIX = "d";
```

Comment:
The prefix to form the key "d:GeneralSupportRequest" for when joining wit the Resources